### PR TITLE
[CP-1807] Create contact based on message with invalid phone number

### PR DESCRIPTION
**Pull request by patryk-sierzega**
Jira: [CP-1807]

- disabled create new contact in thread lists
- disabled create new contact in thread details

[CP-1807]: https://appnroll.atlassian.net/browse/CP-1807?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/messages/helpers/threads.helpers.ts [hunk 1]
```diff
@@ -79,15 +80,19 @@ const isContactWithAnyNumber = (
...
+    isPhoneNumberNotEmpty(contact.secondaryPhoneNumber)
   )
 }
 
-const isPhoneNumberValid = (phoneNumber = ""): phoneNumber is string => {
+const isPhoneNumberNotEmpty = (phoneNumber = ""): phoneNumber is string => {
   return phoneNumber !== ""
 }
 
+export const isPhoneNumberValid = (phoneNumber: string): boolean => {
```
Comment (dkarski) [inline packages/app/src/messages/helpers/threads.helpers.ts:92]: Are you sure this change won't make a regression? Can we catch up 15 minutes before the daily so you can help me understand what it's for? 🙏

Comment (patryk-sierzega) [reply packages/app/src/messages/helpers/threads.helpers.ts:92]: sure, so 10:30 :)

Comment (dkarski) [reply packages/app/src/messages/helpers/threads.helpers.ts:92]: Everything is git. My mistake in code review. I'm sorry! 🙏

**Review by dkarski — APPROVED** [review]
LGTM ✅

**Review by lkowalczyk87 — APPROVED** [review]

**Pull request merged**